src/components/SlateEditor/plugins/definitionList/__tests__/normalizer-test.ts

```diff
@@ -0,0 +1,255 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { createEditor, Descendant, Editor } from 'slate';
+import { withHistory } from 'slate-history';
+import { withReact } from 'slate-react';
+import { plugins } from '../../../../../containers/ArticlePage/LearningResourcePage/components/LearningResourceContent';
+import withPlugins from '../../../utils/withPlugins';
+import { definitionList } from '../utils/defaultBlocks';
+import { TYPE_DEFINTION_DESCRIPTION, TYPE_DEFINTION_LIST, TYPE_DEFINTION_TERM } from '../types';
+import { TYPE_SECTION } from '../../section/types';
+import { TYPE_PARAGRAPH } from '../../paragraph/types';
+
+const editor = withHistory(
+  withReact(withPlugins(createEditor(), plugins('nb', 'nb', { current: () => {} }))),
+);
+
+describe('definition normalizing tests', () => {
+  test('expect list with 1 definition pair not to change', () => {
```

Comment:
Kan godt ha litt mer lesbar test-beskrivelse her, men det har ikke noe å si.
Hvis describe er `definition list normalization` kan test være `should not change a list with 1 definition pair`